daemon/cmd/daemon.go

```diff
@@ -1159,6 +1168,43 @@ func (d *Daemon) bootstrapClusterMesh(nodeMngr *nodemanager.Manager) {
 	bootstrapStats.clusterMeshInit.End(true)
 }
 
+// ReloadOnDeviceChange regenerates device related information and reloads the datapath.
+func (d *Daemon) ReloadOnDeviceChange(devices []string) {
```

Comment:
Please document whether `devices` here represents the new devices, the deleted devices, or the complete set of devices that now exists.

Reply:
Documented.